index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Basic Calculator</title>
+    <link rel="stylesheet" href="style.css">
+    <script type="text/javascript" src="script.js"></script>
+</head>
+<body>
+    <h1>Web Calculator</h1>
+    <h2>Enter Two Numbers To Calculate</h2>
+    <input class="box" type="number" name="firstBox" id="number">
+<br />
+    <input class="box" type="number" name="secondBox" id="number1">
+<br />
+    <input class="box" type="number" name="thirdBox" id="result" readonly>
+    <input class="click" type="button" name="click" value="calculate" onclick="calculate()">
+    <select name="signs" id="signs" onchange="change()">   
```

Comment:
Nowhere in your script file is there a function called "change". It is causing an error every time you change the value. Take this out.